PyHT6022/LibUsbScope.py

```diff
@@ -3,6 +3,7 @@
 import time
 import array
 import usb1
+import libusb1
```

Comment:
Very minor remark: Latest release of python-libusb1 should remove the need to import libusb1 in addition to usb1 (everything needed should be available at usb1 level).
In the case of this patch, `libusb1.LIBUSB_TRANSFER_TYPE_MASK` could be replaced with `usb1.TRANSFER_TYPE_MASK` and `libusb1.LIBUSB_TRANSFER_TYPE_ISOCHRONOUS` with `usb1.TRANSFER_TYPE_ISOCHRONOUS` .
Anyway, as long as python-libusb1 is 1.x (and 2.x is not on the radar yet), I'll retain compatibility so importing libusb1 will work.